micro-ui/web/micro-ui-internals/packages/modules/sandbox/src/configs/UICustomizations.js

```
@@ -91,182 +91,182 @@
           };
-          return (
-            <div>
-              <SVG.Edit style={{ cursor: "pointer" }} onClick={(e) => handleRedirect(e, row?.code, row?.type)} />
-            </div>
-          );
-        default:
-          return t("ES_COMMON_NA");
-      }
+        };
+  
+        delete data.body.custom;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)